keyboards/handwired/dactyl_manuform/6x6/keymaps/happysalada/config.h

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
Technically, this file isn't needed at all.  

serial is the default transport method if there is no define. And the same with master left.  So omitting this file would have the same functionality.

Reply:
Thanks!